### PR TITLE
updated ui and add association.

### PR DESCRIPTION
**Pull request by rishabhharod**

### CODE REVIEW

File: app/controllers/wishlist_items_controller.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def create
     if @wishlist_items.new_record?
       @wishlist_items.save
       flash[:notice] = 'Product wishlisted'
-      redirect_to wishlist_path
+      redirect_to wishlist_path(current_user)
```
Comment (studentrohit123) [inline app/controllers/wishlist_items_controller.rb:10]: you are alredy use this path redirect_to wishlist_path

File: app/views/addresses/index.html.erb [hunk 2]
```diff
@@ -5,125 +5,99 @@
...
+						<% if current_user.addresses.present? %>
+							<div>
+
+							</div>
+							<!-- Button trigger modal -->
+							<button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#exampleModal">
+								Edit form1
+							</button>
+							<!-- Modal -->
+							<%= render partial:"modal" %>
```
Comment (studentrohit123) [inline app/views/addresses/index.html.erb:23]: It is right way to render a modal

**Review by studentrohit123 — COMMENTED** [review]
I have reviewed the code

File: app/views/addresses/index.html.erb (hunk 2 above)
Comment (rishabhharod) [reply app/views/addresses/index.html.erb:23]: yes, this the right way to render the modal but if you the better way pls let me know

File: app/controllers/wishlist_items_controller.rb (hunk 1 above)
Comment (rishabhharod) [reply app/controllers/wishlist_items_controller.rb:10]: check carefully, I'm not repeating the code it's been overwritten

**Pull request merged**